### DISCUSSION TITLE
Sensors unavailable

### DISCUSSION THREAD

**Discussion by GregEdge** [WillCodeForCats/solaredge-modbus-multi#266] [General]
I'm wanting to add some automation but the sensors I need are unavailable. How can I correct this?
![Screenshot_20230221_091132_Home Assistant](https://user-images.githubusercontent.com/121265127/220206094-ffb82ec6-baa7-42bb-93df-3c93df5f42fc.jpg)

**Comment by WillCodeForCats** [comment]
Not all inverters support battery data over modbus even if you have batteries attached and recognized in SetApp. Battery data over modbus is a function that can be enabled or disabled by SolarEdge support. Some inverters have it enabled, some have it disabled. There is no known pattern which will be which. This also includes all site limit and storage control options, which are part of the same data set as batteries.

https://github.com/WillCodeForCats/solaredge-modbus-multi/wiki/Known-Issues

**Reply by GregEdge** [reply]
Thanks. Disappointing that solaredge do this. I'll keep watching the post #63 and hope for a solution.

**Reply by WillCodeForCats** [reply]
Some people have had success contacting solaredge and getting them enabled. I don't know what the magic words are though.
